canonical-web-and-design/maas-ui
# refactor(proxy): merge the proxy with the main package

## Done

- Merge the proxy workspace with the parent package.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- move your `proxy/.env.local` to the parent dir.
- Check that `yarn ui` and `yarn start` both work.

## Fixes

Fixes: canonical-web-and-design/app-tribe#1011.
